src/searchbar/SearchBar-default.js

```diff
@@ -42,10 +42,10 @@ class Search extends Component {
     this.getRef() && this.getRef().blur();
   }
 
-  clearText() {
+  clear() {
     this.getRef() && this.getRef().clear();
     this.props.onChangeText && this.props.onChangeText('');
-    this.props.onClearText && this.props.onClearText();
+    this.props.onClear && this.props.onClear();
```

Comment:
Can we refractor to use the `defaultProps` pattern?

Reply:
Sure, the default searchbar needs a bigger refactor though

Reply:
Lol maybe just mini for now